modules/gateway.go

```diff
@@ -8,12 +8,8 @@ const (
 	// GatewayDir is the name of the directory used to store the gateway's
 	// persistent data.
 	GatewayDir = "gateway"
-	// WellConnectedThreshold is the number of outbound connections at which the
-	// gateway will not attempt to make new outbound connections.
-	WellConnectedThreshold = 8
 )
 
-// TODO: Move this and its functionality into the gateway package.
```

Comment:
ah mistake. I tried to execute the TODO, then decided it was out-of-scope for this PR. It's on my 'let's do this next PR' list.